test_binary_heap.py

```diff
@@ -138,3 +138,18 @@ def test_multiple_operations_maxheap(self):
         self.assertEqual(11, max_heap.extract_root())
         self.assertEqual([8, 6, 6, 5, 3, 4, 5], max_heap.elements())
         self.assertEqual([3, 4, 5, 5, 6, 6, 8], sorted(max_heap.elements()))
+
+
+class TestHeapSort(TestCase):
+    def setUp(self):
+        self.array1 = [4, 3, 6, 8, 11]
+        self.array2 = [4, 1, 3, 2, 16, 9, 10, 14, 8, 7]
```

Comment:
We can add few more tests with repeating numbers...say (2,3,2,1,1,5,4,4,3) just in case